fhir-model/src/main/java/com/ibm/fhir/model/resource/CompartmentDefinition.java

```diff
@@ -1083,7 +1083,7 @@ protected Builder from(CompartmentDefinition compartmentDefinition) {
             valueSet = "http://hl7.org/fhir/ValueSet/resource-types|4.3.0-CIBUILD"
         )
         @Required
-        private final ResourceType code;
+        private final ResourceTypeCode code;
```

Comment:
The headers.txt needing to be updated.

Reply:
headers.txt was updated in previous commit.  now i pushed an update that sets the 2022 copyright on all remaining generated classes in fhir-model